CONTRIBUTING.md

```diff
@@ -21,3 +21,5 @@ If you added a contributor by mistake, you can remove them in a comment with:
 
 If you are making a pull request on behalf of someone else but you had no part in designing the
 feature, you can remove yourself with the above syntax.
+
+If you'd like to contribute a proposal, we offer guidance on our site's [getting involved](https://open-ui.org/get-involved) page. It details the process  of standardizing a UI component, including researching, and reviewing.
```

Comment:
Just a suggestion with regards to detailing the process, what are your thoughts on the following?

```suggestion
If you'd like to contribute a proposal, we offer guidance on our site's [getting involved](https://open-ui.org/get-involved) page. It details the process of standardizing a UI component, including research, proposals, and review.
```
No worries either way; if we push ahead with what you have, a note that there's currently an extra space between "process" and "of".

Reply:
Thanks! I'll take your suggestion, since it's more thorough